v3.4/glfw/build.go

```diff
@@ -0,0 +1,57 @@
+package glfw
+
+/*
+#cgo CFLAGS: -Iglfw/include -D_GNU_SOURCE
+
+// Windows Build Tags
+// ----------------
+// GLFW Options:
+#cgo windows CFLAGS: -D_GLFW_WIN32 -Iglfw/deps/mingw
+
+// Linker Options:
+#cgo windows LDFLAGS: -lgdi32
+
+#cgo !gles2,windows LDFLAGS: -lopengl32
+#cgo gles2,windows LDFLAGS: -lGLESv2
+
+// Darwin Build Tags
+// ----------------
+// GLFW Options:
+#cgo darwin CFLAGS: -D_GLFW_COCOA -Wno-deprecated-declarations
+
+// Linker Options:
+#cgo darwin LDFLAGS: -framework Cocoa -framework IOKit -framework CoreVideo
+
+#cgo !gles2,darwin LDFLAGS: -framework OpenGL
+#cgo gles2,darwin LDFLAGS: -lGLESv2
+
+// Linux Build Tags
+// ----------------
+// GLFW Options:
+#cgo linux,!wayland CFLAGS: -D_GLFW_X11
+#cgo linux,x11 CFLAGS: -D_GLFW_X11
+#cgo linux,wayland CFLAGS: -D_GLFW_WAYLAND
```

Comment:
it seems that the -D_GNU_SOURCE isn't set here

Reply:
We probably don't want it set but I couldn't compile at all without it.

Reply:
If no response to https://github.com/go-gl/glfw/pull/400#issuecomment-2098661241, I'll just mark this as resolved.